platform/features/imagery/test/controllers/ImageryControllerSpec.js

```diff
@@ -165,7 +166,36 @@ define(
                     );
                 });
 
-                it("unsubscribes when scope is destroyed", function () {
+                it("requests telemetry", function () {
+                    expect(openmct.telemetry.request).toHaveBeenCalledWith(
+                        newDomainObject,
+                        jasmine.any(Object)
+                    );
+                });
+
+                it("listens for bounds event", function () {
+                    expect(openmct.time.on).toHaveBeenCalled();
+                });
+
+                it("recognizes duplicate bounds", function () {
+                    expect(controller.equalBounds({
```

Comment:
you are testing an internal method of the controller when you should be testing the behavior.  The code under test should be treated as a black box, and you should only validate what it does, not how it does it.  

In your test, you should trigger the bounds callback and validate that the controller does not request additional data when the bounds are the same as the last bounds.